lib/linkedin-scraper/profile.rb

```diff
@@ -191,6 +193,7 @@ def get_companies(type)
           company[:description] = node.at(".description").text.gsub(/\s+|\n/, ' ').strip if node.at(".description")
 
           start_date, end_date  = node.at('.experience-date-locale').text.strip.split(" – ") rescue nil
+          company[:duration] = node.at('.experience-date-locale').text[/.*\((.*)\)/,1]
```

Comment:
Line is too long. [86/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.